q2_longitudinal/plugin_setup.py

```diff
@@ -47,26 +51,32 @@
     'replicate_handling': Str % Choices(['error', 'random', 'drop']),
 }
 
-base_parameter_descriptions = {
+basest_parameter_descriptions = {
         'metadata': (
             'Sample metadata containing group_column,  state_column, '
             'individual_id_column, and optionally metric values.'),
+        'individual_id_column': (
+            'Metadata column containing IDs for individual subjects.'),
+}
+
+base_parameter_descriptions = {
+        **basest_parameter_descriptions,
         'state_column': ('Metadata column containing state (e.g., Time) '
                          'across which samples are paired.'),
-        'individual_id_column': (
-            'Metadata column containing subject IDs  to use for pairing '
-            'samples. WARNING: if replicates exist for an individual ID at '
-            'either state_1 or state_2, that subject will be dropped and '
-            'reported in standard output by default. Set '
-            'replicate_handling="random" to instead randomly select one '
-            'member, and use --verbose to list conflicts.'),
         'palette': 'Color palette to use for generating boxplots.',
 }
 
 paired_parameter_descriptions = {
         **base_parameter_descriptions,
         'group_column': (
             'Metadata column on which to separate groups for comparison'),
+        'individual_id_column': (
+            'Metadata column containing subject IDs to use for pairing '
+            'samples. WARNING: if replicates exist for an individual ID at '
+            'either state_1 or state_2, that subject will be dropped and '
+            'reported in standard output by default. Set '
+            'replicate_handling="random" to instead randomly select one '
+            'member, and use --verbose to list conflicts.'),
```

Comment:
`--verbose` is a cli-specific flag, but this method can be run in any of the QIIME 2 interfaces. Perhaps just 

> and run in verbose-mode to list conflicts.

Or something to that end.

Reply:
Thanks for catching. That's actually an old description I should have removed earlier. In actions where replicate handling occurs, a warning now appears in the viz to list samples that were dropped.